--- openquake.engine-3.16.4.orig/openquake/commands/run.py	2023-06-29 06:01:24.000000000 -0300
+++ openquake.engine-3.16.4/openquake/commands/run.py	2023-06-29 10:00:58.248754316 -0300
@@ -22,7 +22,10 @@
 import cProfile
 import warnings
 import getpass
-from pandas.core.common import SettingWithCopyWarning
+try:
+    from pandas.core.common import SettingWithCopyWarning
+except ImportError:
+    from pandas.errors import SettingWithCopyWarning
 
 from openquake.baselib import performance, general
 from openquake.hazardlib import valid
